src/components/InventoryTable/InventoryTable.js

```diff
@@ -343,7 +341,6 @@ InventoryTable.propTypes = {
   tableProps: PropTypes.object,
   isRbacEnabled: PropTypes.bool,
   hasCheckbox: PropTypes.bool,
-  onRowClick: PropTypes.func,
```

Comment:
nit-pick: Since the prop is sent down to EntityTable eventually, I think it makes sense to leave it specified in propTypes here as an indication that InventoryTable generally supports this property.